src/UniversalRewardsDistributor.sol

```diff
@@ -133,7 +133,7 @@ contract UniversalRewardsDistributor is IUniversalRewardsDistributor {
         emit EventsLib.Claimed(account, reward, amount);
     }
 
-    /// @notice Forces update the root of a given distribution.
+    /// @notice Forces update the root of a given distribution (bypassing the timelock).
```

Comment:
```suggestion
    /// @notice Forces update the root of a given distribution (bypassing the timelock if any).
```

Reply:
Even if the timelock is zero, it is bypassed in fact lol